signOut.js

```diff
@@ -17,8 +17,8 @@ export async function onSuccess({ params, record, logger, api }) {
 
 /** @type { ActionOptions } */
 export const options = {
-  actionType: "update",
+  actionType: "custom",
```

Comment:
this doesn't need to be changed, with inputs/outputs project it goes away completely